CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Contribution guidelines
+
+Contributions to the documents hosted in the repository are open.
+
+## Reporting problems and making suggestions
+
+If you find or suspect a problem in any of the document, please file [an issue](https://github.com/KhronosGroup/OpenCL-Docs/issues/new) to discuss the details.
+
+You can also file an issue to:
+- suggest improvements to wording;
+- propose a new functionality or improvement to an existing functionality.
```

Comment:
Minor suggestion: since this is documentation and not source code, consider a different word than "functionality".  Maybe "section" or "document"?

Reply:
I was thinking more along the lines of suggesting an improvement to existing features or new features to facilitate external innovations... but I don't know whether/if the wording reflects that. I am happy to change this to anything more descriptive.